sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/EncryptResult.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Azure.Security.KeyVault.Keys.Cryptography
+{
+    /// <summary>
+    /// Represents information about an encryption operation
+    /// </summary>
+    public struct EncryptResult : IJsonDeserializable
+    {
+        private const string KeyIdPropertyName = "kid";
+        private const string CiphertextPropertyName = "value";
+        private const string IvPropertyName = "iv";
+        private const string AuthenticationDataPropertyName = "aad";
+        private const string AuthenticationTagPropertyName = "tag";
+
+        /// <summary>
+        /// The KeyId of the key used to encrypt
+        /// </summary>
+        public string KeyId { get; internal set; }
+
+        /// <summary>
+        /// The ciphertext that is the result of the encryption
+        /// </summary>
+        public byte[] Ciphertext { get; internal set; }
+
+        /// <summary>
+        /// The initialization vector
+        /// </summary>
+        public byte[] Iv { get; internal set; }
+
+        /// <summary>
+        /// The authentication data
```

Comment:
At some point we should improve docs. For example, this needs to say that the properties can be null